dags/check_brokenreaders.py

```diff
@@ -0,0 +1,81 @@
+from airflow import DAG
+from datetime import datetime, timedelta
+from airflow.operators.python_operator import PythonOperator
+from airflow.hooks.postgres_hook import PostgresHook
+from airflow.hooks.base_hook import BaseHook
+from airflow.contrib.operators.slack_webhook_operator import SlackWebhookOperator
+from psycopg2 import sql
+from psycopg2.extras import execute_values
+from psycopg2 import connect, Error
+import logging
+import configparser
+
+LOGGER = logging.getLogger(__name__)
+logging.basicConfig(level=logging.DEBUG)
```

Comment:
Airflow is already taking care of the `logging` configuration, the code should simply call `logging.info`, etc, further down.

https://stackoverflow.com/a/41841689/4047679